src/Rings/localization_interface.jl

```diff
@@ -383,11 +393,24 @@ canonical_unit(f::LocRingElemType) where {LocRingElemType<:AbsLocalizedRingElem}
 
 characteristic(W::AbsLocalizedRing) = characteristic(base_ring(W))
 
-function Base.show(io::IO, W::AbsLocalizedRing) 
-  print(io, "localization of ") 
-  print(io, base_ring(W))
-  print(io, " at the ")
-  print(io, inverted_set(W))
+function Base.show(io::IO, ::MIME"text/plain", W::AbsLocalizedRing)
+  io = pretty(io)
+  println(io, "Localization", Indent())
+  println(io, "of ", Lowercase(), base_ring(W))
+  print(io, "at ")
+  print(io, Lowercase(), inverted_set(W))
```

Comment:
pretty printing of inverted_set is ok for complements of maximal and prime ideals, but still needs changes for powers/products of elements.